lib/arux_app/api/config.rb

```diff
@@ -6,8 +6,8 @@ class Config
       def initialize(options = {})
         self.auth = options[:auth]
 
-        raise API::InitializerError.new(:auth, "can't be blank") if self.auth.nil?
-        raise API::InitializerError.new(:auth, "must be of class type AruxApp::API::Auth") if !self.auth.is_a?(AruxApp::API::Auth)
```

Comment:
Can we keep these here. There would likely not be any regression. But a lot of distributed apps depend on this gem, and I would rather keep the changes to the scope of what is needed for the support portal updates